06_gpu_and_ml/embeddings/wikipedia/README.md

```diff
@@ -0,0 +1,97 @@
+# Embedding Wikipedia in 15 minutes
+
+This example shows how we can embed the entirety of english wikipedia on Modal in just 15 minutes. We've published a detailed writeup which walks you through the implemenation [here](#todo).
+
+## Description
+
+There are a total of 2 files in this repository
+
+- `download.py` : This showcases how to download the Wikipedia dataset into a `Modal` volume. We can take advantage of `Modal`'s high internet speeds to download large datasets quickly.
+
+- `main.py`: This showcases how to run an embedding job on your downloaded dataset and run a parallelizable job using Modal's inbuilt parallelization abstraction.
+
+## Getting Started
+
+You'll need a few packages to get started - we recommend using a virtual environment to install all of the dependencies listed in the `requirements.txt`
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip3 install -r requirements
```

Comment:
Why is `datasets` and `httpx` necessary?

Could be nice to just have a `pip3 install modal`.

Reply:
(Also if we can't remove the requirements entirely, might need to be `requirements.txt`)